README.md

```diff
@@ -1,2 +1,68 @@
 # Restaurant_homepage
-The main goal of this project is for the student to show the understanding of the benefits of modularized code as well as the ability to set up a javascript application made of different modules using webpack.
+
```

Comment:
## Improve the README
- [ ] The documentation looks great. can we add a screenshot to it to make it more descriptive as inspired by this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/). Feel free to use this [template](https://github.com/microverseinc/readme-template).